features/other_a/autocomplete.feature

```diff
@@ -50,3 +50,40 @@ Scenario: Collection autocomplete shows Collection Title and Name
     And I fill in "collection_names" with "Issue"
   Then I should see "jb_fletcher" in the autocomplete
     And I should see "robert_stack" in the autocomplete
+
+Scenario: Pseuds should be added and removed from autocomplete as they are changed
+  Given I have flushed Redis
+    And I am logged in as "new_user"
+  Then the pseud autocomplete should contain "new_user"
+  When I add the pseud "extra"
+  Then the pseud autocomplete should contain "extra (new_user)"
+  When I change the pseud "extra" to "funny"
+    And I go to my pseuds page
+  Then I should not see "extra"
+    And I should see "funny"
+    And the pseud autocomplete should not contain "extra (new_user)"
+    And the pseud autocomplete should contain "funny (new_user)"
+  When I delete the pseud "funny"
+  Then the pseud autocomplete should not contain "funny (different_user)"
+    And the pseud autocomplete should contain "different_user"
```

Comment:
Ugh. I was running these scenarios by hand (ie specifying the line number of the scenario) while I developed them, so I didn't notice the @wip tag, and then I added that last bit of checking to the first test after the fact and only tested it in the context of the whole feature. I have fixed the tests and also since having the tests not working but apparently running is really confusing I have commented the bad tests out and put in a deferred notice